optimum/exporters/ipex/modeling/xpu/utils.py

```diff
@@ -0,0 +1,9 @@
+
+from intel_extension_for_pytorch.transformers.models.xpu.optimize_transformers.ModuleReplacer import ModuleReplacer
+
+def update_patcher_info_on_xpu(patcher, model_name):
+    patch_info = ModuleReplacer.get_patch_info_from_model(model_name)
```

Comment:
Do you think it would be better to put it into optimum-intel? It will be more accessible for users and developers to know which modules in the model are replaced by Ipex.